crates/sui-graphql-rpc/src/types/date_time.rs

```diff
@@ -1,21 +1,65 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use std::{fmt::Display, str::FromStr};
+
 use async_graphql::*;
+use chrono::{
+    prelude::{DateTime as ChronoDateTime, Utc as ChronoUtc},
+    ParseError as ChronoParseError, TimeZone,
+};
 
-// ISO-8601 Date and Time
-// Encoded as a 64-bit unix timestamp
-struct DateTime(u64);
+// ISO-8601 Date and Time: RFC3339 in UTC
+// YYYY-MM-DDTHH:MM:SS.mmmZ
+// Encoded as a 64-bit unix timestamp nanoseconds
+struct DateTime(i64);
```

Comment:
Out of interest, why store the `i64` instead of a `ChronoDataTime<ChronoUtc>` directly?  It seems like going that extra step to/from nanoseconds is only extra work here, and everything else would be simpler if we didn't do that.

Reply:
Addressed in https://github.com/MystenLabs/sui/pull/13492